File: programa_traducido.c
#include <stdio.h>

int main() {
    int numeroDeElementos, promedio, i, elemento;
    scanf("%d", &numeroDeElementos);
    promedio = 0;
    for (i = 1; i <= numeroDeElementos; i++) {
    scanf("%d", &elemento);
    promedio = promedio + elemento;
    }
    promedio = promedio / numeroDeElementos;
    printf("El promedio es %d\n", promedio);
    return 0;
}

File: programa_traducido.py
# Programa traducido desde pseudocódigo
def main():
    # Variables: numeroDeElementos, promedio, i, elemento
    numeroDeElementos = int(input())
    promedio = 0
    for i in range(1, numeroDeElementos + 1):
        elemento = int(input())
        promedio = promedio + elemento
    promedio = promedio / numeroDeElementos
    print("El promedio es", promedio)

if __name__ == '__main__':
    main()
